modules/appeals_api/spec/support/doc_helpers.rb

```diff
@@ -1,5 +1,9 @@
 # frozen_string_literal: true
 
+# These default values are used when running docs specs alongside other kinds of specs via rspec.
+# When generating docs via rake tasks instead, we get these values from the environment set up in the rake task.
+DEFAULT_CONFIG_VALUES = { api_name: 'decision_reviews', api_version: 'v2' }.freeze
```

Comment:
This special case for `ENV['TEST']` happened silently before, because these values would be used automatically if `ENV['API_NAME']` was unset. Now that `ENV['API_NAME']` is set all the time, this needs to be explicit.